mmv1/products/identityplatform/terraform.yaml

```diff
@@ -109,6 +117,9 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "identity_platform_tenant_basic"
         primary_resource_id: "tenant"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "identity_platform_tenant_monitoring"
```

Comment:
The file below references fields that aren't specified in this example block, causing the error.

The .tf.erb file references org_id & billing_acct

Reply:
Check, thanks - fixed.